.env.local

```diff
@@ -101,3 +101,5 @@ ITW_IDP_HINT_TEST=YES
 ITW_IPZS_PRIVACY_URL='https://io.italia.it/informativa-ipzs'
 # ITW Documents on IO URL
 ITW_DOCUMENTS_ON_IO_URL='https://io.italia.it/documenti-su-io'
+# ITW Minimum Integrity Requirements FAQ URL
+ITW_MINIMUM_INTEGRITY_REQUIREMENTS='https://io.italia.it/documenti-su-io/faq/#n1_12
```

Comment:
Is this something that changes with the current environment? If not, why not using a local const directly in the code instead?

Reply:
Thanks for the suggestion. Fixed in [c7694d0](https://github.com/pagopa/io-app/pull/6547/commits/c7694d0b6568365edf6063b846158df50e044493)